src/gui/creds/webflowcredentials.cpp

```diff
@@ -164,7 +164,7 @@ void WebFlowCredentials::askFromUser() {
         }
 
         QString msg = tr("You have been logged out of your account %1 at %2. Please login again.")
-                          .arg(_user, _account->displayName());
+                          .arg(_account->davUser(), _account->url().toDisplayString());
```

Comment:
instead of `davUser()` it should show the Display Name from server
how nice that we do not store it in the config file
see
![image](https://user-images.githubusercontent.com/168967/194359444-389cbb8d-08b0-4ea3-b5c4-aace482eb3b5.png)
```
0\dav_user=john
0\webflow_user=john
```